src/renderer/src/components/common/Flex/Column.tsx

```
@@ -0,0 +1,17 @@
+import { flex } from '@renderer/utils'
+import styled from 'styled-components'
+import type { FlexProps } from './Flex.type'
+
+const Column = ({ children, gap, justifyContent, alignItems, width, height, style }: FlexProps) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)